tests/test_mprotect.py

```diff
@@ -2,11 +2,13 @@
 
 import pwndbg
 import tests
+from pwndbg.gdblib import arch_mod
 
 MPROTECT_BINARY = tests.binaries.get("mprotect.out")
 
 
 def test_mprotect(start_binary):
+    arch_mod.update()
```

Comment:
FWIW this test could look like this:
```py
SMALL_BINARY = tests.binaries.get("crash_simple.out.hardcoded")


def test_mprotect(start_binary):
    """
    Tests mprotect command
    It will mark some memory as executable, then this binary will print "mprotect_ok"
    """
    start_binary(SMALL_BINARY)

    pc = pwndbg.gdblib.regs.pc
    vm = pwndbg.vmmap.find(pc)

    # Check if we can use mprotect with address provided as value
    # and to set page permissions to RWX
    gdb.execute(
        "mprotect %d 4096 PROT_EXEC|PROT_READ|PROT_WRITE" % pc
    )
    vm = pwndbg.vmmap.find(pc)
    assert vm.read and vm.write and vm.execute

    # Check if we can use mprotect with address provided as register
    # and to set page permissions to none
    gdb.execute("mprotect $pc 0x1000 PROT_NONE")
    vm = pwndbg.vmmap.find(pc)
    assert not (vm.read and vm.write and vm.execute)
```

This way we would also ensure we can pass symbols as addresses.
And if we print out the mprotect result, we should assert it too.